packages/gigamon/changelog.yml

```
@@ -0,0 +1,6 @@
+# newer versions go on top
+- version: "0.1.0"
+  changes:
+    - description: Initial draft of the package
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/1 # FIXME Replace with the real PR link
```

```suggestion
      link: https://github.com/elastic/integrations/pull/10268
```

This should be your pull request link